### ISSUE TITLE
Need for naming convention of cell type terms - urgent

### ISSUE THREAD

**Issue by UlrikeS91** [HumanBrainProject/openMINDS_controlledTerms#257]
There are three open issues for cell types: #149, #150 and #240.

We have been avoiding the discussion around how to enter cell types to the controlledTerms for too long now. @lzehl generally would like to avoid the use of brain regions in the cell type names, which I understand, but there are still issues with this:

1. If users studied different cell types in different brain regions, the combination of adding a brain region and a non-brain region specific cell type, will be ambiguous. 
2. Any available ontology enters the cell types as combinations of brain regions + cell type. I know that we do not need to follow this logic, but since generic cell types barely ever exist, none of our controlledTerms will receive an ontology identifier even though they could have one. 

There was a suggestion to add a small schema that produces the link between brain region and cell type, which could be interesting to follow up on now. 

#149 and #150 are rather old, but still have cell types that should maybe be added, but I cannot handle #240 before we have landed on a final conclusion for the naming convention. We really need to land on something asap.

@tgbugs @apdavison @lzehl (feel free to link other people to this issue as well)

**Comment by ehennestad** [comment]
I would lean towards agreeing with @lzehl that it's better to avoid the brain region when naming cell types.

The way I see it a cell type is just a way to classify cells. The key to classification of a cell has traditionally been functional or morphological, and later molecular/genetic with ever increasing number of potential cell types. But a neuron is a neuron no matter if it lives in the brain or in the periphery. And the same for subclasses of neurons.

I think the suggestion to add a small schema that produces the link between brain region and cell type is key here. 

And it seems like this schema already exists. As far as I can see, cell types are only added to the `tissueSample` or `tissueSampleCollection` (and cell in the ephys module) and the information about anatomical position / region should be a part of tissueSample / cell. 

Actually, I think there should be a core schema called `cell`, and the `cell` schema should at least have properties `cellType` and `anatomicalPosition`. A `tissueSample` would then contains `cells` and not cell types and thus by linkage contain information about what cell types are part of the sample and where they are located.

Also, in terms of a cell types, would it be possible to create a hierarchical schema structure, with different levels? I.e there is the very generic neuron cell type, but there is also for example interneuron, inhibitory neuron and somatostatinExpressingNeuron cell types, where all these are subclasses of each other. 

The `cell` schema could have multiple properties related to how to classifiy it based on classification detail. Inspired by the classification of animals, there could be 'Class',' Order', 'Suborder' where 'Class' could be neuron/glia/endothelial/muscle etc, order could be inhibitory/excitatory and suborder is somatostatinExpressing / calretininExpressing etc. Or maybe it would be better to have 'FunctionalType', 'GeneExpressionType' etc as properties of the cell schema. Also from a graph database perspective this seems better. 

If I go into the knowledge graph to look for interneurons, I would not find somatostatinExpressing cells etc. 

Another point: If cell types should contain the region they originate from, the potential number of different cell types would explode, because where do you draw the border for the granularity of regions? And furthermore if someone recorded very specific cell types in different regions, you might end up with SomatostatinExpressingRetrosplenialCortexLayer4Cells.

Basically, as a key strategy, you **do not** want to combine multiple different classification levels and types into specific types because a) you end up with infinitely many combinations and b) its harder to make to search search at different levels of details in the knowledge graph.

**Comment by ehennestad** [comment]
This is a very good reference for the topic of classification of neurons: https://www.nature.com/articles/nrn.2017.85.pdf
Especially, the points made that classifications might have to be based on multiple criteria and that a hierarchical classification system has several advantages. See figure 6 for a nice overview

**Comment by ehennestad** [comment]
This is pretty advanced, but I post it here for reference:
https://www.biorxiv.org/content/10.1101/2021.10.10.463703v2.full.pdf

**Comment by tgbugs** [comment]
See also the NPO paper https://doi.org/10.1007/s12021-022-09566-7 (bioarxiv https://doi.org/10.1101/2020.09.01.278879).

**Comment by ehennestad** [comment]
Super cool cell type explorer (Allen Cell Type Knowledge Explorer)
https://knowledge.brain-map.org/celltypes/

This is based on the Brain Data Standards Ontology which is published on bioRxiv:
https://www.biorxiv.org/content/10.1101/2021.10.10.463703v1.full

Seems like this is now such a extensive resource that maybe it makes sense to include this ontology in openMINDS to describe cell types? 

Another very relevant paper, [BRAIN Initiative Cell Census Network (BICCN)](https://www.nature.com/articles/s41586-021-03950-0#group-1)
https://www.nature.com/articles/s41586-021-03950-0

**Comment by lzehl** [comment]
TODOs (UCSD meeting 2022-11-17)

- [ ] cholinergic interneuron has to be potentially updated (wait for feedback from Interlex)
- [ ] granule cell: change name to granule neuron (move cell to synonym)
- [ ] interneuron: ilx ID -> http://uri.interlex.org/base/ilx_0105664 (use intrinsic neuron as synonym)
- [ ]
